JoinColony/colonyCDapp
# Maintenance: Remove Colony Objective widget and remove functionality to manage Colony objectives

## Description

![remove-col-objectives](https://github.com/user-attachments/assets/c9c57769-ef20-4c3e-a60a-f4328edc8e05)

## Testing

> [!IMPORTANT]
> Check out to another branch and create a Manage Objective action then come back to this branch once that's done.

### Testing steps
1. Please verify that you are no longer able to view the Colony Objectives widget from:
  - Dashboard: `/{colony}`
  - Colony Details page: `/{colonyName}/details`
2. Verify that the Manage Colony Objectives option is not present in the Action Sidebar's Action type field
3. Go to your Dashboard
4. Observe the Latest activity table
5. Verify that the copy for "Manage objective by {user}" is still in tact
6. Filter the activity table using the Colony Objective Action type
7. Verify that the Colony Objective Actions are correctly being filtered
8. Click this Colony Objective Action
9. Verify that the Completed Action form comes up and shows you the details for the Manage objective action
10. On the Completed Action form, verify that the copy for "Manage objective by {user}" is still in tact
11. Open the UserHub and go to the Transactions Tab
12. Verify that you can still see the Edit Colony Details and the copy is in tact
13. Click Edit Colony Details
14. Verify that the Completed Action form comes up and shows you the details for the Manage objective action

Resolves #2797
